src/components/organisms/Footer/Links.module.css

```diff
@@ -0,0 +1,46 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
![grafik](https://user-images.githubusercontent.com/28757404/155016204-1a10a2a2-af37-4a4f-badd-9338c863297a.png)

suggestion/opinion(non-blocking): I think I'd like a `flex-start` styling more on mobile as well. It can look a little weird on tablet sized screens, but I prefer it over the stretched variant we get with `space-between`